implementation/README.md

```diff
@@ -0,0 +1,147 @@
+The aim of LinuxBoot is to reduce complexity and obscure firmware by moving that functionality into kernel and userspace. This chapter describes the procedures from a [LinuxBoot workshop](https://docs.google.com/presentation/d/1s9ka4v7leKeJa3116AQoNb9cv3OqmnW6pgn0ov9WiHo/edit?ts=5e2b227b#slide=id.g7ceec54197_4_163) where an Atomic Pi board with UEFI firmware was converted to run LinuxBoot. 
+
+## A quick refresher on UEFI
+
+UEFI has three sections:
+
++   SEC ("Boot") 
++   PEI ("Very early chip setup and DRAM programming")
++   DXE ("DRAM code")
+
```

Comment:
off the top of my head, I'm inclined to leave lines 11-18 out. But let's talk. It's all true and all factual but there's the question of whether we need to hammer UEFI any harder. Line 19 can definitely stay in. Thoughts?

Reply:
I agree. We want to emphasize what users get with LinuxBoot and lines 11-18 don't add to that.

Reply:
I deleted lines 11-18.